model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1087,6 +1087,44 @@ message SideInput {
   FunctionSpec window_mapping_fn = 3;
 }
 
+message StandardArtifacts {
+  enum Types {
+    // A URN for artifacts stored in a local directory.
+    // payload: ArtifactFilePayload.
+    FILE     = 0 [(beam_urn) = "beam:artifact:file:v1"];
+    // A URN for artifacts embedded in ArtifactInformation proto.
+    // payload: raw data bytes.
+    EMBEDDED = 1 [(beam_urn) = "beam:artifact:embedded:v1"];
+    // A URN for artifacts described by HTTP links.
+    // payload: a string for an artifact HTTP URL
+    HTTP     = 2 [(beam_urn) = "beam:artifact:http:v1"];
+    // A URN for artifacts hosted on PYPI.
+    // artifact_id: a PYPI project name
+    // version_range: a PYPI compatible version string
+    // payload: None
+    PYPI     = 3 [(beam_urn) = "beam:artifact:pypi:v1"];
+    // A URN for artifacts hosted on Maven central.
+    // artifact_id: [maven group id]:[maven artifact id]
+    // version_range: a Maven compatible version string
+    // payload: None
+    MAVEN    = 4 [(beam_urn) = "beam:artifact:maven:v1"];
+  }
+}
+
+message ArtifactFilePayload {
+  // A path to an artifact file on a local system.
+  string local_path = 1;
+  // A generated staged name (no path).
+  string staged_name = 2;
+}
+
+message ArtifactInformation {
+  string urn = 1;
```

Comment:
Looks like `urn` and `payload` combination is mostly common in `beam_runner_api.proto` since I couldn't find any other usage of `string type =`. I think it would be better to follow the same convention here. WDYT?